examples/vue-demo-store-strapi/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "example-vue-demo-store-strapi",
```

Comment:
Pls consider a better name, also for the future. Folder also ... `cms-strapi` so later we have all cms integration closely together ... `cms-storyblok`, `cms-prismic` all below examples folder.

And why the package name contains example and vue-demo store?